apps/meteor/server/models/raw/Rooms.ts

```diff
@@ -868,6 +868,27 @@ export class RoomsRaw extends BaseRaw<IRoom> implements IRoomsModel {
 		return this.col.countDocuments({ federated: true });
 	}
 
+	async countMessagesInActiveFederatedRooms(): Promise<number> {
+		const pipeline = [
+			{
+				$match: {
+					federated: true,
+					$or: [{ archived: false }, { archived: { $exists: false } }],
```

Comment:
do we need the `exists: false` for archived? Can we change it for something like `archived: $ne: true`? would that have the same result?

Reply:
Yes, it would.... much better! Thanks man